exampleSite/config.toml

```diff
@@ -38,6 +38,7 @@ theme="bootstrap-bp-hugo-theme"
 [params]
   description = "Text about the site"
   #images = ["logo.jpg"]
+  # copyright_years = "2018 - 2024"
```

Comment:
Sorry could you please use copyrightYears to make it consistent with the other vars.

Reply:
Sure. Are you fine with a force-push?

Reply:
done.